### PR TITLE
Refs #29212 - install en_US.utf-8 locale when required

### PR DESCRIPTION
**Pull request by wbclark**

### CODE REVIEW

File: manifests/install.pp [hunk 1]
```diff
@@ -1,6 +1,13 @@
 # Install the needed packages for foreman
 class foreman::install {
 
+  if $foreman::db_manage and $facts['os']['family'] == 'RedHat' {
```
Comment (wbclark) [inline manifests/install.pp:4]: I think it's pretty simple to re-work this for other OSes as well if needed, but what about other locales? We seem to be declaring `postgresql::server::db` with `en_US.utf8` hardcoded so I think I can ignore them?

**Comment by wbclark** [comment]
The package exists for EL8 only, causing the acceptance test to fail on EL8. I'm not aware on EL7 equivalent so I can add an additional condition to only install it on EL8.

@ekohl given this, do you still think it makes sense to install it in `manifests/install.pp` as opposed to only in the EL8 acceptance test environment?

File: manifests/install.pp (hunk 1 above)
Comment (ekohl) [reply manifests/install.pp:4]: I agree. The DB is defined as en_US so this needs to be present.

File: manifests/install.pp [hunk 2]
```diff
@@ -1,6 +1,13 @@
...
+    package { 'glibc-langpack-en':
+      ensure => installed,
+      before => Class['foreman::database'],
```
Comment (ekohl) [inline manifests/install.pp:7]: This is a bit redundant since init.pp does the same thing.

**Comment by ekohl** [comment]
https://github.com/theforeman/foreman-packaging/pull/5171 does this in packaging and might be easier.

File: manifests/install.pp [hunk 3]
```diff
@@ -1,6 +1,13 @@
 # Install the needed packages for foreman
 class foreman::install {
 
+  if $foreman::db_manage and $facts['os']['family'] == 'RedHat' {
+    package { 'glibc-langpack-en':
```
Comment (evgeni) [inline manifests/install.pp:5]: AFAIK this package does not exist on EL7, so you'd need to guard that somewhere.

**Comment by evgeni** [comment]
> [theforeman/foreman-packaging#5171](https://github.com/theforeman/foreman-packaging/pull/5171) does this in packaging and might be easier.

So I've been wondering the same. Especially as the installer needs to be run with the right locale too (and I think we force it somewhere), so the package needs to be present *before* the installer runs?

**Comment by evgeni** [comment]
That said, it definitely won't hurt.

**Comment by ekohl** [comment]
> Especially as the installer needs to be run with the right locale too (and I think we force it somewhere), so the package needs to be present _before_ the installer runs?

It needs to be present before the database is created.

**Comment by wbclark** [comment]
Closing in favor of https://github.com/theforeman/foreman-packaging/pull/5171

**Pull request closed without merging**